### PR TITLE
Added test for extension.ts 

### PR DESCRIPTION
**Pull request by sudhirverma**
Issue #76

### CODE REVIEW

**Comment by mohitsuman** [comment]
testPR

**Comment by mohitsuman** [comment]
@sudhirverma can you check and fix the scenarios where build is failing for Windows.

**Comment by mohitsuman** [comment]
@sudhirverma Please squash all commits into one and push it and amend the commit message for sake of clarity.

**Comment by mohitsuman** [comment]
testPR

**Comment by sudhirverma** [comment]
testPR

**Comment by sudhirverma** [comment]
testPR

**Comment by sudhirverma** [comment]
@dgolovin @jrichter1 Please review this pr

File: test/extension.test.ts [hunk 1]
```diff
@@ -1,13 +1,105 @@
...
+
+    teardown(() => {
+        sandbox.restore();
+    });
+
+    test('Extension should be present', () => {
+        assert.ok(vscode.extensions.getExtension('redhat.vscode-adapters'));
+    });
+
+    test('Server is started at extension activation time', async () => {
```
Comment (jrichter1) [inline test/extension.test.ts]: This test case doesn't assert that server is started

**Comment by jrichter1** [comment]
Also, how come so much of package-lock is being changed without a single change in package.json?

**Comment by sudhirverma** [comment]
@jrichter1 can you review this pr

**Comment by dgolovin** [comment]
@jrichter1 when transitive dependencies have open version range, then anything can change in that package-lock.file, that is why I hate it when I do `git diff` after `npm install`.

**Pull request merged**